### PR TITLE
First iteration of UI development process

### PR DESCRIPTION
**Pull request by stugoo**

### CODE REVIEW

File: docs/_product/guidelines/development-process.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+3. Document your component to the below standard
+4. Test your component across browsers, devices and viewports for stability and accessibility
+5. Publish your component to the style guide
+6. Integrate your component into the application
+7. Communicate that the new component has been released
+
+### Naming a component
+Naming things is hard, so it’s important that if you’re unable to make a decision on a component name, just pick one and move forward. If you’re unsure, it’s likely the name will change at a later stage in development or review. The code should be architected in a way that’s easy to change.
+
+With the exception of icons*, components should be named to reflect their specific functional use. Whilst presentational naming should not be used, there are always cases where this is unavoidable. Presentational naming should be limited to branding heavy component variations, such as banners, billboards and components that rely on branding imagery.
```
Comment (jamesmockett) [inline docs/_product/guidelines/development-process.md]: Is the asterisk after 'icons' meant to be a footnote reference, or a typo?

File: docs/_product/guidelines/development-process.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+- Broadcast new component
+
+# FAQs
+## Should I add a new CSS file to the application?
+If you’re writing new components then these should be written using SCSS in the UI development standard. However you should not add a new CSS bundle to the application. If there is a proper use case to add a new file please speak to #ui-engineering
+
+## My component is only used once or is not really part of FreeStyle, what should I do?
+You should still follow the development and deployment process, treating it the same as other widely used components, because your component is only being used once now, in future it may find a new home. Every component should be documented to the same standard as reliability and consistency in the design system is paramount.
+
+# My component needs to be broken down into sub-components do I need to follow this process for every part?
```
Comment (jamesmockett) [inline docs/_product/guidelines/development-process.md]: This has a different heading level to the other FAQs.

**Comment by chrisgreen1993** [comment]
This may be a silly question, but I thought we were getting rid of origin?

**Comment by stugoo** [comment]
@chrisgreen1993 Welcome to the party! This is where the FreeStyle docs currently live... Ideally we'd look to get rid of this in the application, alas here we are

**Pull request merged**